src/styles/sidebar/components/annotation-user.scss

```diff
@@ -0,0 +1,14 @@
+.annotation-user,
+.annotation-user a {
+  @include font-normal;
+  color: $grey-7;
+  font-weight: bold;
+
+  .is_dimmed & {
```

Comment:
`is-{state}` / `has-{attribute}` classes have a hyphen in the name rather than an underscore.